.github/workflows/ci.yml

```diff
@@ -8,6 +8,7 @@ on:
       - '.github/workflows/ci.yml'
       - 'MANIFEST.in'
       - 'pyproject.toml'
+      - 'reframe_config.py'
```

Comment:
Don't you need the whole path `benchmarks/reframe_config.py` here?

Reply:
Hmm, I would think so, but it triggered without. Confusing

Ah, it triggered because the workflow itself changed